### PR TITLE
Documenting Current and Improved OSCAL Issue Triage Process

### LINKED ISSUE #1633: Documenting and Improving OSCAL Issue Triage
### User Story

As a NIST OSCAL developer or community member, in order to understand the lifecycle of a question I pose, an issue I raise, or a code modification I propose, I want to see explanation of the triage process the NIST Team performs, if any, and know how to track the status of one of these interactions I have with the team.

### Goals

- [x] Document triage process for the whole NIST OSCAL Team
- [x] Consult team to understand, beyond documenting what is done, if there are any improvements to make
- [x] Review recommendations, circulate them with the NIST OSCAL Team
- ~#1712~
- ~Make applicable changes~
- ~Socialize final process back to team and the larger community~

### Dependencies

- ~Spike improvements to GitHub Actions to not add new issues to a repo-specific board, but add them all the main project board if possible, [perhaps by switching to this after spiking](https://github.com/actions/add-to-project)~

### Acceptance Criteria

- ~All [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes in this issue have been updated. Changes to the OSCAL website can be made in the docs/content directory of your branch.~
- [x] A Pull Request (PR) is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.
- [x] The CI-CD build process runs without any reported errors on the PR. This can be confirmed by reviewing that all checks have passed in the PR.

### PR DESCRIPTION
**Pull request by aj-stein-nist**
# Committer Notes

Add an ADR to document the current issue triage process and plot a way forward. Closes #1633.

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers
](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- ~Do all automated CI/CD checks pass?~

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- ~Have you written new tests for your core changes, as applicable?~
- ~Have you included examples of how to use your new feature(s)?~
- ~Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.~

### CODE REVIEW

File: decisions/0004-issue-triage.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+The team can be more involved in issue triage as part of [a backlog refinement process](https://www.scrum.org/resources/blog/product-backlog-refinement-explained-13). Currently, the team does not engage in periodic, formalized backlog refinement either. Moving forward, issue triage can be considered the first step to formally adding an issue or rejecting it by closing the issue. If done frequently, issue triage would only be needed periodically for new issues. If there were no new issues queued for triage between refinement sessions, the team could primarily focus on backlog refinement for the next sprint and following sprints.
+
+From a tooling perspective, whereas the newer GitHub Projects service did not have API support for complex operations and GitHub Actions did not readily exist during the migration to the new project board, these conditions are no longer the case.
+
+Therefore issue triage process can be formalized as part of backlog refinment scheduled periodically for some or all team members to be involved for sufficient context. Automation can be updated to have all work requests in issues be track in one location, not four or more.
```
Comment (nikitawootten-nist) [inline decisions/0004-issue-triage.md:48]:
```suggestion
Therefore issue triage process can be formalized as part of backlog refinement, scheduled periodically for some or all team members to be involved for sufficient context. Automation can be updated to have all work requests in issues be tracked in one location, not four or more.
```

**Review by nikitawootten-nist — APPROVED** [review]
I requested some small typo fixes, but I agree wholeheartedly with the conclusion.

File: decisions/0004-issue-triage.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# 4. Issue Triage for OSCAL Core Projects
+
+Date: 2023/03/03
+
+## Status
+
+Feedback Requested
```
Comment (aj-stein-nist) [inline decisions/0004-issue-triage.md:7]: You know, for when you we actually do want to merge with the correct status next time.

```suggestion
Approved
```

**Review by wendellpiez — APPROVED** [review]
Absolutely.

**Review by Compton-NIST — APPROVED** [review]
Always in favor of triage as something for everyone.

**Review by iMichaela — APPROVED** [review]

**Pull request merged**